streampipes-rest/src/main/java/org/apache/streampipes/rest/impl/datalake/DataLakeResource.java

```diff
@@ -416,6 +417,14 @@ private boolean checkProvidedQueryParams(Map<String, String> providedParams) {
   public ResponseEntity<?> setDataLakeRetention(
       @PathVariable String elementId,
       @RequestBody RetentionTimeConfig retention){
+        try {
+          
+          retention.getRetentionExportConfig().getExportProviderSettings().setSecretKey(
+          SecretEncryptionManager.encrypt(retention.getRetentionExportConfig().getExportProviderSettings().getSecretKey()));
```

Comment:
If we update the RetentionTimeConfig and users do not change the secret key, will the secret be encrypted twice?